examples/hotelling_law/hotelling_law/model.py

```diff
@@ -211,10 +211,10 @@ def step(self):
 
     def recalculate_market_share(self):
         # Reset market share for all stores directly
-        for store in self.store_agents:
+        for store in  self.agents_by_type[StoreAgent]:
             store.market_share = 0
 
-        for consumer in self.consumer_agents:
+        for consumer in  self.agents_by_type[ConsumerAgent]:
```

Comment:
Is there a space too much here?

Reply:
I'll run ruff once I am done and fix it all